### DISCUSSION TITLE
Formatting not working correctly

### DISCUSSION THREAD

**Discussion by d3buggerdan** [facebook/lexical#6838] [Q&A]
For some reason with my editor, the bold and italics formats work fine but formats such as underline and strikethrough don't? I tried using setFormat and I check using the hasFormat which returns true but nothing visual is shown to reflect it in the editor. Looking for any fixes thanks.

**Comment by etrepum** [comment]
Configure your theme accordingly, formatting is applied by classes from the theme. You can use the playground's css as a reference.
